services.yaml

```diff
@@ -1753,13 +1753,47 @@ paths:
     post:
       tags:
         - Events
-      summary: Force Event Snapshot
+      summary: Force event snapshot
       description: Force snapshot on the next time the event occurs
       responses:
         200:
           description: OK
         401:
           $ref: "#/definitions/UnauthorizedError"
+  /services/{service_id}/events/{event_id}/actions:
+    get:
+      tags:
+        - Events
+      summary: "Fetch event actions"
+      description: "Fetch event actions"
+      parameters:
+          - in: path
+            name: service_id
+            description: Environment to use
+            type: string
+            required: true
+          - in: path
+            name: event_id
+            description: Event to use
+            type: string
+            required: true
+          - in: query
+            name: type
+            description: Filter by specific type
+            type: string
+            required: false
+      responses:
+        200:
+          description: Event actions
+          schema:
+            actions:
+              type: "array"
+              items:
+                $ref: "#/definitions/EventAction"
+        401:
+          $ref: "#/definitions/UnauthorizedError"
```

Comment:
please merge the volumetrics as it contains a lot of the fixes we need such as having errors as "#/responses/.." and not definitions